src/transformers/models/maskformer/__init__.py

```diff
@@ -43,7 +43,7 @@
 else:
     _import_structure["modeling_maskformer"] = [
         "MASKFORMER_PRETRAINED_MODEL_ARCHIVE_LIST",
-        "MaskFormerForInstanceSegmentation",
+        "MaskFormerForUniversalSegmentation",
```

Comment:
Shouldn't we keep `MaskFormerForInstanceSegmentation` until it's fully deprecated?

```suggestion
        "MaskFormerForInstanceSegmentation",
        "MaskFormerForUniversalSegmentation",
```